EnglishFilter/sections/adservers.txt

```diff
@@ -162,6 +162,8 @@
 !
 ||asgclickkl.com^$third-party
 ||displaynetworkprofit.com^$third-party
+||xxutoyreg.com^$third-party
+||btemegxbnalq.com^$third-party
```

Comment:
As far as I understand, these rules should block popups, if so then it stopped working or at least it doesn't work on my end.

Please add `wordcounter.icu` to this rule:
https://github.com/AdguardTeam/AdguardFilters/blob/e059d24e9fb50d4ed6f294ae4a1789aa55bb0bc5/EnglishFilter/sections/general_extensions.txt#L36
and also to this one:
https://github.com/AdguardTeam/AdguardFilters/blob/e059d24e9fb50d4ed6f294ae4a1789aa55bb0bc5/EnglishFilter/sections/general_extensions.txt#L3702
so these popups should be blocked even if "ad server" will be changed.